.github/workflows/ci.yml

```diff
@@ -3,15 +3,6 @@ name: Ubuntu
 on: [push, pull_request]
 
 jobs:
-  bionic-ci:
-    runs-on: ubuntu-latest
-    name: Ubuntu Bionic CI
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Compile and test
-        id: ci
-        uses: ignition-tooling/action-ignition-ci@bionic
```

Comment:
Do we need to add `cpplint-enabled` and `cppcheck-enabled` to `focal-ci`?

Reply:
What I'm doing with this series of PRs is to pass to Focal whatever was enabled on Bionic. Since SDFormat didn't have anything enabled on Bionic, I didn't add anything to Focal. It's possible that enabling new checks will require changes in the code, so I think that could come in a separate PR.

Reply:
Sounds good. Didn't know if those were enabled by default on `bionic`.

Reply:
Ah gotcha. They're all opt-in. I was actually surprised to see that SDFormat had codecov enabled on Focal, all other libraries had it on Bionic.

Reply:
On bionic, it looks like `make codecheck` was on by default:
https://github.com/ignition-tooling/action-ignition-ci/blob/c859eba535d1cde8dd5b6c3175d48d7af1b2f04c/entrypoint.sh#L140-L149

Reply:
ohhh you're right! I didn't realize this was different between the branches. I'll need to revisit all other PRs. Thanks for digging in.